plugins/resources-generator/src/main/kotlin/dev/icerock/gradle/generator/apple/AppleGeneratorHelper.kt

```diff
@@ -2,22 +2,22 @@
  * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
  */
 
-package dev.icerock.gradle.generator.ios
+package dev.icerock.gradle.generator.apple
 
 import com.squareup.kotlinpoet.ClassName
 import com.squareup.kotlinpoet.KModifier
 import com.squareup.kotlinpoet.PropertySpec
 import com.squareup.kotlinpoet.TypeSpec
 import dev.icerock.gradle.generator.ObjectBodyExtendable
 
-class IosGeneratorHelper(private val propertyName: String = "nsBundle") :
+class AppleGeneratorHelper(private val propertyName: String = "nsBundle") :
     ObjectBodyExtendable {
 
     override fun extendObjectBody(classBuilder: TypeSpec.Builder) {
         val bundleType = ClassName("platform.Foundation", "NSBundle")
```

Comment:
it's valid for all apple platforms? ios, macos, watchos, tvos? :)

Reply:
That's right, I was thinking a bit forward though :D.
Also I couldn't really come up with a better name for iOS + macOS